### DISCUSSION TITLE
APK version of LMS ?

### DISCUSSION THREAD

**Discussion by joeymard** [LMS-Community/slimserver#1158] [Ideas]
Would it be possible to obtain an APK version of LMS in order to install it in an android device ? Thanks in advance

**Comment by michaelherger** [comment]
See https://forums.slimdevices.com/forum/user-forums/general-discussion/1674045-announcement-running-lms-on-an-android-device#post1674045.

**Reply by joeymard** [reply]
Thank you !!!
